core/Measurable/Type/Manager.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+namespace Piwik\Measurable\Type;
+
+use Piwik\Plugin\Manager as PluginManager;
+use Piwik\Measurable\Type;
+
+class Manager
```

Comment:
I will rename it but we should discuss this in general at some point. I don't mind writing the `as` statements. `MeasurableTypeManager` then we can also just avoid namespaces and put everything in one folder :)